README.md

```diff
@@ -22,7 +22,9 @@ Run the build:
 docker run -it --rm -v $(pwd)/:/home/ --name microvisor-gpio-sample microvisor-gpio-sample-image
 ```
 
-The bundle for submission to Twilio for deployment will be `build/Demo/gpio_toggle_demo.zip`.
+Two bundles will be generated for submission to Twilio:
+ - `build/Demo/gpio_toggle_demo.debug.bundle` with remote debugging support
```

Comment:
This worked for me :+1: the quickstart guide uses the old zip name so will need updating to align

Reply:
Why have .zip for non-debug bundles and .bundle for debug bundles. These should both be .zip as they are the same thing apart from their contents, no? Adding .debug into the file name is fine, I think.

Reply:
My idea was that user is never supposed to extract these, that bundles are in *.zip format is an implementation detail which doesn't necessarily need to be exposed.